File: php/classic_puzzle/easy/onboarding.php
<?php
// game loop
while (TRUE)
{
    fscanf(STDIN, "%s",
        $enemy1 // name of enemy 1
    );
    fscanf(STDIN, "%d",
        $dist1 // distance to enemy 1
    );
    fscanf(STDIN, "%s",
        $enemy2 // name of enemy 2
    );
    fscanf(STDIN, "%d",
        $dist2 // distance to enemy 2
    );

    // Write an action using echo(). DON'T FORGET THE TRAILING \n

    // Enter the code here
    if ($dist1 < $dist2) {
        echo ($enemy1."\n");
    } else {
        echo ($enemy2."\n");
    }
}
?>

File: kotlin/practice/classic_puzzle/easy/onboarding.kt
import java.util.*
import java.io.*
import java.math.*

fun main(args : Array<String>) {
    val input = Scanner(System.`in`)

    // game loop
    while (true) {
        val enemy1 = input.next() // name of enemy 1
        val dist1 = input.nextInt() // distance to enemy 1
        val enemy2 = input.next() // name of enemy 2
        val dist2 = input.nextInt() // distance to enemy 2

        // Write an action using println()
        if (dist1 < dist2) {
            println(enemy1)
        } else {
            println(enemy2)
        }

        // Enter the code here

    }
}
